### DISCUSSION TITLE
I need help on how to change from southern hemisphere back to northern hemisphere

### DISCUSSION THREAD

**Discussion by durdetoetuka** [Monika-After-Story/MonikaModDev#9604] [QA - tech support + general] [answered]
The first time monika asked me if I live in the southern or northern hemisphere and I acidentally clicked on southern, now i need help to change it back please

**Comment by Iceninja64** [comment, marked as answer]
Go back to that same topic and press the correct hemisphere. You should be able to find it in the search box
